src/MyList.js

```diff
@@ -0,0 +1,7 @@
+function MyList() {
```

Comment:
Do you still need this file?

Reply:
![1555](https://user-images.githubusercontent.com/60723932/172860154-e52ff6b9-d3c8-448e-87e7-760558a56915.PNG)

Reply:
![image](https://user-images.githubusercontent.com/60723932/172859015-ec64520f-a9cd-4c04-bca6-0713afbb9be8.png)

Reply:
click on Manage and Trust this file

Reply:
![image](https://user-images.githubusercontent.com/60723932/172932404-e18c904c-a4de-464e-8232-9813fd3743a3.png)

Reply:
as you can see the folder is trusted but still i can not call EntryList file in App.js file

Reply:
what do you mean I can't call EntryList file in App.js

Reply:
I was trying to to write 
import Entry ....etc 

but I was able to write till import E

after many tires also i could not
so I wrote the call line in a notepad file and copy and paste is in visual studio edit